src/bindings/python/setup.cfg

```diff
@@ -83,3 +83,4 @@ disallow_untyped_defs = True
 disallow_untyped_calls = True
 check_untyped_defs = True
 show_none_errors  = True
+implicit_optional = True
```

Comment:
will discard it after https://github.com/openvinotoolkit/openvino/pull/13913 is merged